<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Farmopolis</class>
 <widget class="QMainWindow" name="Farmopolis">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>549</width>
    <height>468</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_24">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_2" rowstretch="2,20,2" columnstretch="1,22,1">
      <item row="1" column="0">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="2">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0" colspan="3">
       <spacer name="verticalSpacer_4">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="1">
       <widget class="QWidget" name="farmopoliswidget" native="true">
        <layout class="QGridLayout" name="gridLayout_23">
         <property name="leftMargin">
          <number>30</number>
         </property>
         <property name="topMargin">
          <number>30</number>
         </property>
         <property name="rightMargin">
          <number>30</number>
         </property>
         <property name="bottomMargin">
          <number>30</number>
         </property>
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout_18" rowstretch="2,8,14" columnstretch="8,16">
           <property name="spacing">
            <number>30</number>
           </property>
           <item row="2" column="0">
            <spacer name="verticalSpacer_6">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="1" column="1" rowspan="2">
            <widget class="QWidget" name="widget" native="true">
             <layout class="QGridLayout" name="gridLayout_26">
              <property name="leftMargin">
               <number>9</number>
              </property>
              <property name="topMargin">
               <number>9</number>
              </property>
              <property name="rightMargin">
               <number>9</number>
              </property>
              <property name="bottomMargin">
               <number>9</number>
              </property>
              <item row="0" column="0">
               <layout class="QGridLayout" name="gridLayout_25">
                <item row="0" column="0">
                 <widget class="QWidget" name="groundwidget" native="true"/>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item row="0" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout">
             <property name="spacing">
              <number>10</number>
             </property>
             <item>
              <widget class="QPushButton" name="roundStartpushButton">
               <property name="text">
                <string>start</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="useractivelabel">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="activeuserlabel">
               <property name="text">
                <string>: user1</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_4">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item row="1" column="0">
            <widget class="QWidget" name="productsstorewidget" native="true">
             <layout class="QGridLayout" name="gridLayout_19">
              <property name="leftMargin">
               <number>30</number>
              </property>
              <property name="topMargin">
               <number>30</number>
              </property>
              <property name="rightMargin">
               <number>30</number>
              </property>
              <property name="bottomMargin">
               <number>30</number>
              </property>
              <item row="0" column="0">
               <layout class="QGridLayout" name="gridLayout_20">
                <item row="1" column="3">
                 <widget class="QLabel" name="salespricehenlabel">
                  <property name="text">
                   <string>     3</string>
                  </property>
                 </widget>
                </item>
                <item row="4" column="0">
                 <widget class="QLabel" name="wheatNamelabel">
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item row="6" column="1">
                 <widget class="QLabel" name="purchasepriceWorkerlabel">
                  <property name="text">
                   <string>     5</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="henNamelabel">
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item row="5" column="0">
                 <widget class="QLabel" name="barleyNamelabel">
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item row="2" column="4">
                 <widget class="QPushButton" name="buysheeppushButton">
                  <property name="text">
                   <string>buy</string>
                  </property>
                 </widget>
                </item>
                <item row="6" column="0">
                 <widget class="QLabel" name="workerNamelabel">
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item row="3" column="3">
                 <widget class="QLabel" name="salespricecowlabel">
                  <property name="text">
                   <string>     5</string>
                  </property>
                 </widget>
                </item>
                <item row="5" column="1">
                 <widget class="QLabel" name="purchasepricebarleylabel">
                  <property name="text">
                   <string>     2</string>
                  </property>
                 </widget>
                </item>
                <item row="4" column="2">
                 <widget class="QLabel" name="productiontimewheatlabel">
                  <property name="text">
                   <string>   25 s </string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="3">
                 <widget class="QLabel" name="salespricesheeplabel">
                  <property name="text">
                   <string>     3</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="2">
                 <widget class="QLabel" name="productiontimesheeplabel">
                  <property name="text">
                   <string>   15 s </string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="3">
                 <widget class="QLabel" name="salesPricelabel">
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item row="4" column="4">
                 <widget class="QPushButton" name="buywheatpushButton">
                  <property name="text">
                   <string>buy</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="4">
                 <widget class="QPushButton" name="buyhenpushButton">
                  <property name="text">
                   <string>buy</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="4">
                 <widget class="QPushButton" name="buycowpushButton">
                  <property name="text">
                   <string>buy</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QLabel" name="purchasePricelabel">
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item row="5" column="4">
                 <widget class="QPushButton" name="buybarleypushButton">
                  <property name="text">
                   <string>buy</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QLabel" name="sheepNamelabel">
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item row="0" column="2">
                 <widget class="QLabel" name="productionTimelabel">
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item row="4" column="3">
                 <widget class="QLabel" name="salespricewheatlabel">
                  <property name="text">
                   <string>     3</string>
                  </property>
                 </widget>
                </item>
                <item row="5" column="3">
                 <widget class="QLabel" name="salespricebarleylabel">
                  <property name="text">
                   <string>     2</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QLabel" name="purchasepricehenlabel">
                  <property name="text">
                   <string>     3</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1">
                 <widget class="QLabel" name="purchasepricesheeplabel">
                  <property name="text">
                   <string>     5</string>
                  </property>
                 </widget>
                </item>
                <item row="4" column="1">
                 <widget class="QLabel" name="purchasepricewheatlabel">
                  <property name="text">
                   <string>     2</string>
                  </property>
                 </widget>
                </item>
                <item row="5" column="2">
                 <widget class="QLabel" name="productiontimebarleylabel">
                  <property name="text">
                   <string>   20 s </string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="2">
                 <widget class="QLabel" name="productiontimecowlabel">
                  <property name="text">
                   <string>   20 s </string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="1">
                 <widget class="QLabel" name="purchasepricecowlabel">
                  <property name="text">
                   <string>     7</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="0">
                 <widget class="QLabel" name="productNamelabe">
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item row="3" column="0">
                 <widget class="QLabel" name="cowNamelabel">
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item row="1" column="2">
                 <widget class="QLabel" name="productiontimehenlabel">
                  <property name="text">
                   <string>   20 s </string>
                  </property>
                 </widget>
                </item>
                <item row="6" column="2" colspan="3">
                 <widget class="QPushButton" name="buyWorkerpushButton">
                  <property name="text">
                   <string>buy</string>
                  </property>
                 </widget>
                </item>
                <item row="7" column="1">
                 <widget class="QLabel" name="purchasepricelabel">
                  <property name="text">
                   <string>     3</string>
                  </property>
                 </widget>
                </item>
                <item row="7" column="0">
                 <widget class="QLabel" name="groundNamelabel">
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item row="7" column="2" colspan="3">
                 <widget class="QPushButton" name="buygroundpushButton">
                  <property name="text">
                   <string>buy</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item row="0" column="1">
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <property name="spacing">
              <number>10</number>
             </property>
             <item>
              <widget class="QLabel" name="numberOfWorkersIconlabel">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="numberOfWorkerslabel">
               <property name="text">
                <string>1 / 1</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="coinInventoryIconlabel">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="coinInventorylabel">
               <property name="text">
                <string>10 </string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="0" colspan="3">
       <spacer name="verticalSpacer_5">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>549</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
